app/graphql/schema/queries/customer-queries-by-sbi.js

```diff
@@ -1,28 +1,52 @@
 const cosmos = require('../../../cosmos')
 const { cosmosConfig } = require('../../../config')
+const { convertCosmosTimestamp } = require('../../../utils')
 
-const customerQueriesBySbi = async (_root, args, context) => {
-  const { queriesDatabase } = await cosmos()
-  const querySpec = {
-    query: 'SELECT * FROM customerQueries cq WHERE cq.sbi = @sbi',
-    parameters: [{ name: '@sbi', value: `${args.sbi}` }]
-  }
-  const response = await queriesDatabase
-    .container(cosmosConfig.queriesContainer)
-    .items.query(querySpec)
-    .fetchAll()
-  return {
-    sbi: args.sbi,
-    customerQueries: response.resources.map((x) => ({
-      id: x.id,
-      crn: x.crn,
-      sbi: x.sbi,
-      heading: x.heading,
-      body: x.body
-    }))
+const customerQueryTicketsBySbi = async (_root, args, context) => {
+  try {
+    const { queriesDatabase } = await cosmos()
+
+    const querySpec = {
+      query: 'SELECT * FROM customerQueryResponse cq WHERE cq.sbi = @sbi ORDER BY cq._ts DESC',
+      parameters: [{ name: '@sbi', value: `${args.sbi}` }]
+    }
+
+    const response = await queriesDatabase
+      .container(cosmosConfig.queriesContainer)
+      .items.query(querySpec)
+      .fetchAll()
+
+    if (!args.sbi) {
+      throw new Error('SBI must be provided')
+    }
+
+    if (!response.resources.length) {
+      throw new Error(`No customer query data found for SBI ${args.sbi}`)
+    }
+
+    return {
+      status: {
```

Comment:
As below, these are hard coded

Reply:
See above comment, this has been removed now.